presto-testing/src/main/java/io/prestosql/testing/datatype/DataTypeTest.java

```diff
@@ -90,9 +146,14 @@ Object toPrestoQueryResult()
             return dataType.toPrestoQueryResult(value);
         }
 
-        String toLiteral()
+        public String toLiteral()
```

Comment:
intentional? (same for other methods)

Reply:
Yes, I wanted to be able to create new DataSetup implementations in a different package.

Reply:
Yes, I wanted to be able to create new DataSetup implementations in different packages.